README.md

```diff
@@ -253,10 +258,12 @@ server running on port 8080
 Here,
 
 - `--jsonrpc-url` indicates the blockchain node for the state resolver to contact in order to get the latest state to verify the proof responses against
-- `--state-contract` indicates the `State.sol` contract address that captures the issuer's latest state
+- `--state-contract` indicates the `State.sol` contract address that captures the issuer's latest state. (you can find the value of it in the `state` field in `~/iden3/deploy_output.json` file)
```

Comment:
It would be nice if the script could read the RPC URL and contract address directly from `~/iden3/deploy_output.json`.

Reply:
The URL can be obtained from hre.network.config.url.

Reply:
The URL can be obtained from hre.network.config.url.

Reply:
If the code is bundled into a microservice, those two values are likely to be configurations.

But indeed, it would be handy for trying out this repo if it reads from the local files.